packages/core/src/controllers/verify.ts

```diff
@@ -33,7 +38,23 @@ export class Verify extends IVerify {
       this.removeIframe();
     }
     this.verifyUrl = verifyUrl;
-    await this.createIframe();
+
+    await this.createIframe().catch((error) => {
+      this.logger.error(`Verify iframe failed to load: ${this.verifyUrl}`);
+      this.logger.error(error);
+    });
+
+    if (this.initialized) return;
```

Comment:
So `this.initialized` is a proxy for the initial `createIframe` having succeeded on `.com`? Wouldn't it be easier to do the handling below in the `catch` of the first attempt and avoid this check on `this.initialized`? 

E.g. `.warn` on initial failure and then log error like [here](https://github.com/WalletConnect/walletconnect-monorepo/pull/3410/files#diff-adfd0925fb8e8e5ac3dfdf97b5e40ddf5957b1bc86ef2e065bc88f60d887ea82R53) once the fallback fails too.

At least that's how we're doing it for the WS connection atm.

Reply:
It's done so intentionally as we have to catch both attempts (main url & fallback) unlike in the WS connection where the fallback rejection is unhandled on purpose